### ISSUE TITLE
is it possible to do.....

### ISSUE THREAD

**Issue by lollerollo** [blydro/whatsapp-track#1]
Hi blydro, I have tried your script and I really appreciate it.
So, from what I saw the script works on the person I had already chatted together ( A )
I would like to know if you think could be possible to make the script work and taking number from
the list that appears when is pressed the "New message Button" ( B )

![wa_1](https://user-images.githubusercontent.com/35402080/35487971-4a13ae10-0482-11e8-94c9-3e387511dc89.jpg)

Thank you very much!!!

**Comment by blydro** [comment]
Yes I think this is definitely possible, I need to snoop around the Store object some more. `Store.Contact` contains many more contacts than `Store.Presence`, but I haven't yet found a way to subscribe to these contacts. 

If you or anyone else has any ideas...
